src/components/icon/icon-request.ts

```diff
@@ -47,6 +47,8 @@ export const getSvgContent = (
                   return svgContent;
                 });
               }
+              // FIXME check this returned value
+              return new Promise<string>(() => '');
```

Comment:
Can you try this:
```suggestion
              throw new Error('Failed to load icon ' + namespace + ':' + name);
```
(better use template strings :P)